tests/main/services-snapctl/task.yaml

```diff
@@ -76,7 +76,7 @@ execute: |
 
     echo "Reinstalling the snap with configure hook calling snapctl restart works"
     snap set test-snapd-service command=restart
-    "$TESTSTOOLS"/snaps-state install-local test-snapd-service
+    "$TESTSTOOLS"/snaps-state install-local test-snapd-service --ignore-running
```

Comment:
it would be good to have a comment that explains what is running that needs to be ignored

Reply:
Added